src/components/SocialMedia.tsx

```diff
@@ -7,16 +7,18 @@ type DisplayMode = "icon" | "text";
 export interface ISocialMediaProps {
   position?: Position;
   mode?: DisplayMode;
+  className?: string;
 }
 
 const SocialMedia: FunctionComponent<ISocialMediaProps> = ({
+  className,
   position,
   mode,
 }) => {
   return (
     <ul
       className={`social-media ${position === "row" ? "position-row" : ""}
-      ${mode === "text" ? "mode-text" : ""}`}
+      ${mode === "text" ? "mode-text" : ""} ${className}`}

```

Comment:
Check what value has `${className}` in case this property not defined? (empty string or "undefined").

Reply:
- [ ] Done.